monitoredresources/gcp_metadata_config.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitoredresources
+
+import (
+	"cloud.google.com/go/compute/metadata"
+	"os"
+	"strings"
+)
+
+// gcpMetadataConfigMap holds mapping of Attribute Name and Attribute Value
+// in GKE container and GCE instance environment
+var gcpMetadataConfigMap map[string]string
```

Comment:
why are we sharing this globally not just returning it from the function?

Reply:
I will ping you on hangout about this.